internal/probe/discriminate.go

```
@@ -0,0 +1,716 @@
+			landsExtreme = m.to >= hi-landing
+		}
+		if mag >= qualMag && landsExtreme {
+			hasCliff = true
+			cliffMove = math.Max(cliffMove, mag)
+		} else {
+			runnerUp = math.Max(runnerUp, mag)
+		}
+	}
+	if hasCliff {
+		v.Finding = FindingCliff
+		v.Confidence = clamp01(1 - math.Pow(runnerUp/cliffMove, 2))
```

golangci-lint (2.12.2)

[error] QF1005: could expand call to math.Pow

(staticcheck)